tests/wpt/mozilla/tests/css/background_border_padding_crash.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+    <title>Assure that different writing modes do not crash background building in Servo</title>
```

Comment:
This isn't run as a test. This needs a testharness `<script>` and something like:

```
async_test(function(t) {
    window.onload = t.step_func_done();
}, "Didn't crash");
```

Reply:
Can we write it as a reference test instead?

Reply:
@jdm How do I best convert this to a reference test as I don't have a reference file?

Reply:
Wouldn't the reference file look like ` <span>Filler Text Filler Text</span>`?